content/docs/dvclive/get-started.md

```diff
@@ -20,11 +20,30 @@ See [`Live()`](/doc/dvclive/api-reference/live) for details.
 
 ### Log data
 
+- Scalar
+
 ```python
 live.log("acc", 0.9)
 ```
 
-See `Live.log()` for details.
```

Comment:
It would be good to get other another opinion on whether these examples of logging other data types make this too long. @jorgeorpinel I know you have a lot of other reviews, but any thoughts on this in particular? 🙏

Reply:
Does it make sense to use tabs here?

Reply:
Tried tabs and I kind of like it (should be visible in latest deployment)